jest.config.js

```diff
@@ -0,0 +1,16 @@
+/** @type {import('ts-jest/dist/types').InitialOptionsTsJest} */
+module.exports = {
+  roots: [
+    'src/tests'
+  ],
+  setupFilesAfterEnv: [
+    './src/tests/jest-setup.ts'
+  ],
+  preset: 'ts-jest/presets/js-with-babel',
+  testEnvironment: 'jsdom',
+  globals: {
+    'ts-jest': {
+      tsconfig: './tsconfig.json'
```

Comment:
While tsconfig options can be added inline, we reference a tsconfig file since eslint also needs to see it.